resources/Subgraph/Mainnet-Voting-Entities.md

```diff
@@ -0,0 +1,180 @@
+| id              | ID!    | Utility entity that links data from a single ethereum address. Id of the entity is the ethereum address itself                             |
+| address         | Bytes! |                                                                                                                                            |
+| countReveals    | BigInt | Number of price requests that this user has revealed a vote for, and therefore participated in as a voter                                  |
+| countRetrievals | BigInt | Provides a lower-bound on # of votes that user has correctly voted for. User may not have retrieved rewards for all of their correct votes |
+| votesCommited   | Int!   |                                                                                                                                            |
+
+## Collateral
+
+Description:
+
+| Field         | Type       | Description                                                                                                                                     |
+| ------------- | ---------- | ----------------------------------------------------------------------------------------------------------------------------------------------- |
+| id            | ID!        | Represents approved collateral that is whitelisted in the AddressWhitelist and whose fees are set in the Store. Id of the entity is its address |
+| decimals      | Int!       |                                                                                                                                                 |
+| name          | String!    |                                                                                                                                                 |
+| symbol        | String!    |                                                                                                                                                 |
+| isOnWhitelist | Boolean!   | Is token currently whitelisted as collateral                                                                                                    |
+| finalFee      | BigDecimal |
+
+## PriceIdentifier
+
+Description:
+
+| Field         | Type                             | Description                                                                                                                                                 |
+| ------------- | -------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| id            | ID!                              |                                                                                                                                                             |

```

Comment:
Yes, they ideally would all have descriptions but it would take me awhile to get used to Uma in order to enter all of those.  Is there a way to work together to fill those in?

Reply:
I think it's fine for now, we can start with this and progressively enhance.